### PR TITLE
Update Spanish and French translations

### PR DESCRIPTION
**Pull request by franglais125**

### CODE REVIEW

**Comment by franglais125** [comment]
@zapashcanon would you mind checking the French translation? [968b7f439a639023402c47cfdc63a366cfd56537]

File: po/fr.po [hunk 1]
```diff
@@ -66,234 +70,295 @@ msgid "Border width"
...
+msgid "Show open windows previews."
+msgstr "Afficher des aperçus de fenêtres."
+
+#: Settings.ui.h:40
 msgid ""
-"If disabled, these settings are acccessible from gnome-tweak-tool or the "
+"If disabled, these settings are accessible from gnome-tweak-tool or the "
 "extension website."
 msgstr ""
 "Si désactivés, ces paramètres sont acccessibles via gnome-tweak-tool ou le "
```
Comment (zapashcanon) [inline po/fr.po]: `accessibles` ? :)

File: po/fr.po [hunk 2]
```diff
@@ -66,234 +70,295 @@ msgid "Border width"
...
+#: Settings.ui.h:44
+msgid "Launchers"
+msgstr "Lanceurs"
+
+#: Settings.ui.h:45
+msgid ""
+"Enable Super+(0-9) as shortcuts to activate apps. It can also be used "
+"together with Shift and Ctrl."
+msgstr ""
+"Activer Super+(0-9) comme raccourcis pour activers les applications. On "
```
Comment (zapashcanon) [inline po/fr.po]: `activer`

**Comment by franglais125** [comment]
Merci, that was fast! Embarrassing that I got those typos in there.

**Comment by zapashcanon** [comment]
Looks like the first one was here before. ;)

**Comment by micheleg** [comment]
Thanks, merged [abfe5dff0bf9b, 9eec8d9889f].

**Pull request merged**